tests/test_api.py

```diff
@@ -576,9 +575,61 @@ def test_get_datatype_manifest(self, client, syn_token):
                 "Year of Birth": "Int64",
                 "entityId": "string"}
 
+    # small manifest: syn51078535; big manifest: syn51156998
+    @pytest.mark.parametrize("manifest_id",["syn51156998", "syn51156998"]) 
```

Comment:
TIL (today I learned): I didn't realize you could chain these - which makes it more readable, but see my comment about `parametrize` below